TimeCalc.java

```diff
@@ -1,5 +1,26 @@
 public class TimeCalc {
     public static void main(String[] args) {
-        // Replace this comment with your code
+        String time = args[0];
+        int minutesToAdd = Integer.parseInt(args[1]);
+        int hours = Integer.parseInt(time.substring(0,2));
+        int minutes = Integer.parseInt(time.substring(3,5));
+        minutes = (hours * 60) + minutes +  minutesToAdd;

```

Comment:
it is better to declare new variables for this.. not tamper with value given on input unless told otherwise